docs/data/zod.data.ts

```diff
@@ -0,0 +1,24 @@
+import { type ZodTypeAny } from 'zod';
+import { defineLoader } from 'vitepress';
+import { printNode, zodToTs } from 'zod-to-ts';
+
+import { configSchema, type Config } from '../../src/core/schemas.js';
+
+type Data = {
+  configSchema: string;
+};
+
+function convertZodSchemaToString(schema: ZodTypeAny | Config, schemaTitle: string) {
```

Comment:
I assume it should work to just use `typeof configSchema` and we should also be able to get rid of configSchema as unknown as Config afterwards.

Reply:
due to zodToTs is first parameter type ZodTypeAny, like this? 
```suggestion
function convertZodSchemaToString(schema: ZodTypeAny | typeof configSchema, schemaTitle: string) { 
const { node } = zodToTs(schema as ZodTypeAny, schemaTitle);
  return printNode(node);
}
```